tests/unit/test_log_proxy_consumer.py

```diff
@@ -1,6 +1,7 @@
 # Copyright 2020 Canonical Ltd.
```

Comment:
It would. I'd rather handle as a follow-up, since this PR is also reasonably urgent, as it includes fixes for `LogProxyConsumer` being broken since yesterday.

@dstathis @Abuelodelanada ?